site/docs/build-ref.html

```diff
@@ -357,6 +357,29 @@ <h4>Package names, <code>//<b>package-name</b>:...</code></h4>
   end with a slash.
 </p>
 
+<h4 id="pattern_expansion">Target pattern expansions, <code>//<b>...</b></code>
+  and <code>//<b>:all</b></code></h4>
+<p>
+  The two <code>...</code>and <code>:all</code> notations are special when used
+  on the command line. These target patterns are expanded by bazel before the
+  build is started.
+</p>
+<p>
+  Adding <code>...</code> after a package tells bazel that you're interested in
+  all targets of all packages below the specified directory. For example
+  <code>//...</code> means "all targets in all packages in the source tree".
+  On the other hand <code>//foo/bar/...</code> means "all targets in all
+  packages in the <code>//foo/bar</code> package and all packages underneath
+  the <code>//foo/bar</code> package.
+</p>
+<p>
+  Adding <code>:all</code> after a package tells bazel that you're interested
+  in all targets within that package. For example <code>//:all</code> means
```

Comment:
There's some nuance here. See the ref in https://docs.bazel.build/versions/4.0.0/guide.html#specifying-targets-to-build. I don't think it's worth getting too deep in the weeds in this text. But technically `:all` doesn't cover all targets.